src/components/RenderBooks.js

```diff
@@ -0,0 +1,38 @@
+import '../styles/RenderBooks.css';
+
+function RenderBooks() {
+  return (
```

Comment:
- [ ] For rendering a single book, Kindly use the created component `Book.js` and pass the `title` and `author` as props to it.